src/pal/src/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ endif()
 
 # Include directories
 
-include_directories(include)
+include_directories(include ${CoreCLR_SOURCE_DIR}/src/inc)
```

Comment:
I don't see the CoreCLR_SOURCE_DIR defined anywhere. The variable you want is CLR_DIR.

Reply:
CMAKE defines CoreCLR_SOURCE_DIR because of the project(CoreCLR) statement in the root directory.